backend/chainlit/playground/providers/openai.py

```diff
@@ -145,8 +145,11 @@ async def create_completion(self, request):
 
         async def create_event_stream():
             async for stream_resp in response:
-                token = stream_resp.choices[0]["delta"].get("content", "")
-                yield token
+                if 'choices' in stream_resp and len(stream_resp.choices) > 0:
```

Comment:
one question here, should we not use 
```py
hasattr(stream_resp, 'choices')
```
since `choices` is an attribute and not dict key?